docker_templates/partials/header.j2

```diff
@@ -0,0 +1,5 @@
+ENV JAVA_HOME /opt/java/openjdk
+ENV PATH $JAVA_HOME/bin:$PATH
+
+# Default to UTF-8 file.encoding
+ENV LANG='en_US.UTF-8' LANGUAGE='en_US:en' LC_ALL='en_US.UTF-8'
```

Comment:
This partials file might better be called something like `nix-env.j2`?

Reply:
done